src/pages/RoomDescriptionPage.tsx

```diff
@@ -62,6 +63,19 @@ function RoomDescriptionPage({report, policies}: RoomDescriptionPageProps) {
         goBack();
     }, [report.reportID, report.description, description, goBack]);
 
+    const validate = useCallback((): Errors => {
+        const errors: Errors = {};
+        const descriptionLength = description.trim().length;
+        if (descriptionLength > CONST.REPORT_DESCRIPTION.MAX_LENGTH) {
```

Comment:
This PR caused a bug. https://github.com/Expensify/App/issues/58850#issuecomment-2741568091

state is only updated after the validation.